essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,154 +1,208 @@
-# \[Draft] How to Contribute to Appflowy
-
+# \[Draft] How to Contribute to AppFlowy
 
 <!-- TABLE OF CONTENTS -->
+
 <details>
   <summary>Table of Contents</summary>
   <ol>
     <li>
       <a href="#introduction">Introduction</a>
       <ul>
-        <li><a href="#general-purpose-of-this-article">General Purpose of This Article</a></li>
+        <li><a href="#overview">Overview</a></li>
       </ul>
     </li>
     <li>
       <a href="#how-to-contribute-to-appflowy">How to Contribute to Appflowy</a>
       <ul>
         <li><a href="#how-to-connect-with-the-team">How to Connect With the Team</a></li>
-        <li><a href="#how-to-identify-and-solve-your-first-pr">How to Identify and Solve Your First PR</a></li>
+        <li><a href="#how-to-identify-and-resolve-your-first-pr">How to Identify and Resolve Your First PR</a></li>
```

Comment:
`solve` and `resolve` both sound a bit off, what about `How to identify and contribute with your first PR` or similar?

A bug can be resolved, but a feature request can be implemented.

Reply:
changed it "open" as per your other suggestions